types/mithril/index.d.ts

```diff
@@ -136,11 +135,13 @@ declare namespace Mithril {
 		useBody?: boolean;
 		/** If false, redraws mounted components upon completion of the request. If true, it does not. */
 		background?: boolean;
+		/** Milliseconds a request can take before automatically being terminated. */
+		timeout?: number;
 	}
 
 	interface JsonpOptions {
 		/** The data to be interpolated into the URL and serialized into the querystring. */
-		data?: any;
+		params?: { [id: string]: any };
```

Comment:
Don't forget `body:` (unless of course you have it specified elsewhere).

Reply:
Okay, good catch.